src/main/java/pl/betoncraft/betonquest/events/ChatEvent.java

```diff
@@ -18,7 +18,7 @@ public ChatEvent(final Instruction instruction) throws InstructionParseException
         try {
             final String string = instruction.getInstruction();
             messages = string.trim().substring(string.indexOf(" ") + 1).split("\\|");
```

Comment:
If the `String.trim()` call is moved up an `IndexOutOfBoundsException` becomes impossible. Instead  an `InstructionParseException` should be thrown if there is no space (and thus no actual message) in the instruction string (after trimming).
```suggestion
            final String string = instruction.getInstruction().trim();
            messages = string.substring(string.indexOf(" ") + 1).split("\\|");
```